app/src/main/java/org/oppia/app/topic/TopicActivityController.kt

```diff
@@ -0,0 +1,18 @@
+package org.oppia.app.topic
+
+import androidx.appcompat.app.AppCompatActivity
+import org.oppia.app.R
+import org.oppia.app.activity.ActivityScope
+import javax.inject.Inject
+
+/** The controller for [TopicActivity]. */
+@ActivityScope
+class TopicActivityController @Inject constructor(private val activity: AppCompatActivity) {
+  fun handleOnCreate() {
+    activity.setContentView(R.layout.topic_activity)
+    activity.supportFragmentManager.beginTransaction().add(
```

Comment:
Should we be checking first whether the topic fragment is already added?

Reply:
Added code for checking topic-fragment, so as to not re-create same fragment.